arango/client.py

```diff
@@ -213,6 +218,17 @@ def db(
                 deserializer=self._deserializer,
                 superuser_token=superuser_token,
             )
+        elif user_token is not None:
+            connection = JwtConnection(
+                hosts=self._hosts,
+                host_resolver=self._host_resolver,
+                sessions=self._sessions,
+                db_name=name,
+                http_client=self._http,
+                serializer=self._serializer,
+                deserializer=self._deserializer,
+                user_token=user_token,
+            )
```

Comment:
Curious whether we want to support the possibility of passing a `user_token` **and** and a `username/password` combination? This way a user could rely on the automatic `db.conn.refresh_token` instead of manually calling `db.conn.set_token`

Reply:
Right now we already have that functionality if `auth_method="jwt"` is passed?

Reply:
Currently if `user_token` and `username/password` is passed (regardless if `auth_method="jwt"` or not), we create a `JwtConnection` object with only `user_token`.

https://github.com/ArangoDB-Community/python-arango/blob/34aaa138e62dace4469c3da6874adba4625f3ef0/arango/client.py#L210-L257

The quick fix to this would be to pass `username/password` into the `user_token`-based `JwtConnection` so that the user can make use of `conn.refresh_token()`, assuming that the `username/password` info is correct

Reply:
resolving after internal discussion:

- `user_token`-based `JwtConnection` remains
- automatic token refresh will only be offered if `username/password` & `auth_method="jwt"` is set
- manual token setter is available for both `JwtConnection` and `JwtSuperuserConnection` objects